demo/admin/src/dam/getRandomUnsplashImage.ts

```diff
@@ -1,8 +1,20 @@
+import { GQLLicenseInput } from "@src/graphql.generated";
+
 export interface UnsplashImage {
     file: File;
     url: string;
 }
 
+class ExtendedFile extends File {
+    license?: GQLLicenseInput;
+    title?: string;
+    altText?: string;
+
+    constructor(blob: BlobPart[], fileName: string, options: FilePropertyBag) {
+        super(blob, fileName, options);
+    }
+}
+
```

Comment:
should this be moved into the package and be imported by the project?

btw. should the demo set randomized license, title, altText